paddle/fluid/operators/compat/fused_conv2d.pbtxt

```diff
@@ -39,14 +39,44 @@ def {
     name: "data_format"
     type: STRING
   }
+}
+extra {
```

Comment:
Shouldn't _conv2d.pbtxt_ and _conv_op.cc_ be also adjusted? Or this will be done in next PR?

Reply:
Yes, this will be done in next PR.